homeassistant/components/bbox/sensor.py

```diff
@@ -59,23 +61,26 @@
             cv.ensure_list, [vol.In(SENSOR_TYPES)]
         ),
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
```

Comment:
This violates both ADR-0007 & ADR-0010.

I do not think we should make an exception to these rules.

Reply:
Completely agree after reading them. My initial thought process was just to "fix" this missing option but I understand that this would lead to even more exceptions which we should not encourage.